config/install/menu_export.export_data.yml

```diff
@@ -4408,13 +4418,110 @@
   id:
     value: '91'
   uuid:
-    value: 999d6544-208e-416b-8fdf-e882cf0f560e
+    value: 174e5eee-e2c0-4eba-a164-84ca70d66ef0
   revision_id:
     value: '91'
   langcode:
     value: en
   bundle:
     value: menu_link_content
+  revision_created:
+    value: '1714982130'
+  revision_user:
+    target_id: '0'
+  revision_log_message: false
+  enabled:
+    value: '1'
+  title:
+    value: 'Academic Sessions List'
+  description: false
+  menu_name:
+    value: state-admin-menu
+  link:
+    uri: 'internal:/academic-sessions'
+    title: ''
+    options:
+      fa_icon: ''
+      fa_icon_prefix: fa
+      fa_icon_tag: i
+      fa_icon_appearance: before
+  external:
+    value: '0'
+  rediscover:
+    value: '1'
+  weight:
+    value: '-50'
+  expanded:
+    value: '0'
+  parent:
+    value: 'menu_link_content:bb3d3819-d081-4a8b-acb2-4b67fba67617'
+  changed:
+    value: '1715049767'
+  default_langcode:
+    value: '1'
+  revision_default:
+    value: '1'
+  revision_translation_affected:
+    value: '1'
+96:
+  id:
+    value: '96'
+  uuid:
+    value: bb3d3819-d081-4a8b-acb2-4b67fba67617
+  revision_id:
+    value: '96'
+  langcode:
+    value: en
+  bundle:
+    value: menu_link_content
+  revision_created:
+    value: '1715049748'
+  revision_user: false
+  revision_log_message: false
+  enabled:
+    value: '1'
+  title:
+    value: 'Academic Sessions'
+  description: false
+  menu_name:
+    value: state-admin-menu
+  link:
+    uri: 'route:<nolink>'
+    title: ''
+    options:
+      fa_icon: ''
+      fa_icon_prefix: fa
+      fa_icon_tag: i
+      fa_icon_appearance: before
+  external:
```

Comment:
Why we are adding the new link here? We already have a menu item. 

![Screenshot from 2024-05-07 14-34-32](https://github.com/Indus-Action-Initiatives/rte-mis/assets/35484116/b2b5e1f2-6b4c-4f14-8133-c07bf5fe0319)

Reply:
Ok so we will be following this. I have added a action link to create a academic session from the listing page itself.